bmlt-meeting-list.php

```diff
@@ -22,9 +22,57 @@ class Bread {
 		var $mpdf = '';
 		var $meeting_count = 0;
 		var $formats_used = '';
+		var $formats_by_key = '';
 		var $formats_spanish = '';
 		var $formats_all = '';
-		
+		var $meeting_fields = array (
+			'id_bigint',
+			'service_body_bigint',
+			'weekday_tinyint',
+			'start_time',
+			'duration_time',
+			'formats',
+			'email_contact',
+			'comments',
+			'location_city_subsection',
+			'location_nation',
+			'location_postal_code_1',
+			'location_province',
+			'location_sub_province',
+			'location_municipality',
+			'location_neighborhood',
+			'location_street',
+			'location_info',
+			'location_text',
+			'meeting_name',
+			'bus_lines',
+			'format_shared_id_list'
+		);
+		var $calculated_fields = array(
+			'duration_m',
+			'duration_h',
+			'day',
+			'day_abbr',
+			'area_name',
+		);	
+		var $synonyms = array (
```

Comment:
might this be better known as `$legacy_synonyms`, since we are keeping this for backward compatibility?